spring-integration-http/src/main/java/org/springframework/integration/http/inbound/HttpRequestHandlingEndpointSupport.java

```diff
@@ -159,6 +160,7 @@ public HttpRequestHandlingEndpointSupport() {
 	}
 
 	public HttpRequestHandlingEndpointSupport(boolean expectReply) {
+		super(expectReply);
```

Comment:
:+1: It actually doesn't matter because we only call `sendAndReceive()` if `expectReply` is true. But I thought it more correct to set it this way rather than always `true`.